app/assets/stylesheets/groceries/_show.css.scss

```diff
@@ -98,14 +98,10 @@
       font-size: 1.2rem;
 
       i {
-        background: $light-gray;
-        padding: 4px 6px;
-        border-radius: 20px;
-        color: white;
-
-        &:hover {
-          background: darken($light-gray, 15%);
-        }
+        font-size: 1.5rem;
+        color: $light-gray;
+        
+        &:hover { color: $red; }
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 5